index.css

```diff
@@ -32,4 +32,9 @@ body{
 
 .centerAlign {
     text-align: center;
+}
+
+.row div{
```

Comment:
cud u change it from using divs to usinga a class name and put the class name inside the divs?